Reflex/make.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable filenames/match-regex */
```

Comment:
We should get rid of this ... ESLint should allow files called "make.js"

Reply:
#20